editor/components/block-settings-menu/block-remove-button.js

```diff
@@ -26,6 +26,8 @@ export function BlockRemoveButton( { onRemove, onClick = noop, isLocked, role }
 			icon="trash"
 			label={ label }
 			role={ role }
```

Comment:
we should use `{ ...props }` instead of explicit props here to allow passing any extra props to the underlying button